model/vm_host.rb

```diff
@@ -195,11 +195,18 @@ def download_boot_image(image_name, version:, custom_url: nil)
   # Introduced for downloading firmware via REPL.
   def download_firmware(version_x64: nil, version_arm64: nil, sha256_x64: nil, sha256_arm64: nil)
     version, sha256 = (arch == "x64") ? [version_x64, sha256_x64] : [version_arm64, sha256_arm64]
-    raise ArgumentError, "No version provided" if version.nil?
-    raise ArgumentError, "No SHA-256 digest provided" if sha256.nil?
+    fail ArgumentError, "No version provided" if version.nil?
+    fail ArgumentError, "No SHA-256 digest provided" if sha256.nil?
```

Comment:
oh huh, did I make this comment about a block not changed by your commits? oops.